src/Tizen.NUI/src/public/WindowSystem/TizenRegion.cs

```diff
@@ -0,0 +1,160 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Tizen.NUI.WindowSystem.Shell
+{
+    /// <summary>
+    /// Tizen Window System Shell.
+    /// This is a basic handle class for others in WsShell.
+    /// Others class in WsShell using this basic class to use TIzen Window System Shell.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
```

Comment:
if you want make this as hidden API by tagging [EditorBrowsable... Never], please remove <since_tizen> tag. as I know, it is for public API.

Reply:
I remove all <since_tizen> tag except public APIs and enums.